src/vcf/report_writer.cpp

```diff
@@ -0,0 +1,120 @@
+//
+// Created by jmmut on 25/05/16.
+//
+
+#include "vcf/report_writer.hpp"
+
+namespace ebi
+{
+  namespace vcf
+  {
+    SqliteReportWriter::SqliteReportWriter(std::string db_name) : db_name{db_name}, current_transaction_size{0},
+                                                      transaction_size{1000000}, sleep_time{500}
+    {
+        int rc = sqlite3_open(db_name.c_str(), &db);
+        if (rc != SQLITE_OK) {
+            sqlite3_close(db);
+            throw std::runtime_error(std::string("Can't open database: ") + sqlite3_errmsg(db));
+        }
+
+        char *zErrMsg = NULL;
+        rc = sqlite3_exec(db, "CREATE TABLE if not exists Errors (line int, message varchar(255));", NULL, 0,
+                          &zErrMsg);
```

Comment:
Table and column names in SQL are normally lowercase to avoid case-sensitivity problems.